include/s3select.h

```diff
@@ -2420,9 +2475,13 @@ class base_s3object
 	projections_resuls.clear();
 	for (auto& i : m_projections)
 	{
+    if(i->is_column_reference())  {
```

Comment:
it is not correct to extract the `is_column_reference` value n this way.
 the `column_reference` is been overriding.
 
 `result_values_to_string` should get several values, one per each projection.